### PR TITLE
rpi-kernel: update to 5.4.59.

### PR DESCRIPTION
**Pull request by pbui**
- Built for armv6l, armv7l, and aarch64.
- Tested on armv6l, armv7l.

### CODE REVIEW

**Comment by pbui** [comment]
@Piraty This works on one of my armv7l rpis... I will test more tomorrow morning.

**Comment by ericonr** [comment]
Have you tested on musl? Shouldn't make a difference, especially because we haven't done the time64 update, but you never know. I can try it on my rpi zero.

**Comment by pbui** [comment]
@ericonr I have not tested with a musl system (I don't have any).  If you could test that out, then that would be great.

File: srcpkgs/rpi-kernel/template [hunk 1]
```diff
@@ -1,23 +1,23 @@
 # Template file for 'rpi-kernel'
 #
 # We track the latest Raspberry Pi LTS kernel as that is what is used in the
-# official Raspbian distribution. This is currently 4.19:
+# official Raspbian distribution. This is currently 5.4:
 #
-#   https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=224931
+#   https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=269769
```
Comment (ericonr) [inline srcpkgs/rpi-kernel/template:6]: You added an accidental whitespace there at the end.

**Comment by ericonr** [comment]
Baisc operation on armv6l-musl is working fine.

**Comment by sgn** [comment]
On 2020-08-23 20:15:54-0700, Peter Bui <notifications@github.com> wrote:
> - Built for armv6l, armv7l, and aarch64.
> - Tested on armv6l, armv7l.

Built and Tested on aarch64-musl. uptime for about 1 days, until now.

-- 
Danh

**Comment by pbui** [comment]
I've been running this for a few days and things appear to be stable.  Should be good to go.

**Comment by Piraty** [comment]
modules (at least wireguard) fail to build due to [60df1aee2aecb53efb4218b4dfdf7d6c80a5a3de](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=60df1aee2aecb53efb4218b4dfdf7d6c80a5a3de)

**Comment by pbui** [comment]
@Piraty @ahesford I was able get `modpost` to build with the following hack to the `dkms` hook:

https://gist.github.com/pbui/a368283bbbf030f01db1d8f1a7d381f6

Basically, I temporarily add back the `mod` target to the `scripts` Makefile and then remove it after it is built.

Ideally, we can invoke `make` with an appropriate target or command-line argument to build the program, but I don't really fully grok the kernel build system, so I don't know what that would be.

**Comment by Piraty** [comment]
working on it

**Pull request closed without merging**